.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Bug report
+about: Tell us about a problem you are experiencing
+
+---
+
+/kind bug
```

Comment:
Should we have a list of options like bug, feature request, question?

Reply:
Make sense. I have added feature request and question template. PTAL.